src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <div id="result-container">
+      <p id="text-result">
+        {result}
+      </p>
+    </div>
+  );
+};
+
+Display.defaultProps = {
+  result: 0,
+};
+Display.propTypes = {
+  result: PropTypes.string,
+};
```

Comment:
from your `propTypes` (line 19) , `result` is a string, this means that line 17 should be
```
result: '0',
```
instead of 
```
result: 0,
```
i guess you can also remove the 'trailing comma' sign at end of this line 😊